libraries/mapr_utils.rb

```diff
@@ -18,6 +18,19 @@ def configured_disks
       disks.sort
     end
 
+    def unpartitioned_disks(node)
+      disks_and_partitions = ::Dir.glob ['/dev/disk/by-id/wwn-*']
+      partition_regex = Regexp.new(/-part[0-9]*/)
+      partitionned_disks = disks_and_partitions.grep(partition_regex).map{|dev| dev.sub(partition_regex, '')}
+      disks = disks_and_partitions.grep_v(partition_regex)
+      unpartitioned = disks - partitionned_disks
+      unpartitioned.select do |wwn|
+        status = node['block_device'][::File.basename(::File.readlink(wwn))]
```

Comment:
How sure you're that this attribute is a generic one (that is available in chef by default) and not that specific to Criteo?

Reply:
`block_device` is listed here: https://docs.chef.io/ohai.html